components/organization/Repository.vue

```diff
@@ -0,0 +1,86 @@
+<template>
+  <div class="box-backdrop">
+    <div class="box">
+      <header class="box-header">
+        <slot name="header">
+          <!--{{repository.name}} -->
+          Client
+        </slot>
+      </header>
+
+      <section class="box-description">
+        <slot name="description">
+          <!-- {{repository.description}} -->
+          UNDER CONSTR Uddd fffdd ddddd dddd d dd ddC T ION
+        </slot>
+      </section>
+
+      <section class="box-languages">
+        <slot name="languages">
+          <!--   
+            {{repoditory.languange}}
+            -->
+          Javascript | Vue | HTML
+        </slot>
+      </section>
+
+      <footer class="box-footer">
+        <slot name="footer">
+          <img width="20px" height="20px" src="..\..\static\fork.png" /> 5
+          <pre></pre>
+          <img width="18px" height="18px" src="..\..\static\star.png" /> 10
```

Comment:
Use the [element](https://element.eleme.io/#/en-US/component/icon) icons when needed:

```suggestion
          <i class="el-icon-star-on"/> 10
```